README.md

```diff
@@ -10,7 +10,7 @@ A [node.js](https://nodejs.org) >= 16.12.0 setup with [pnpm](https://pnpm.io) is
 ### 📦 Install dependencies
 
 ```bash
-$ pnpm install
+$ npm i pnpm
```

Comment:
this appears to be an invalid changes. as https://github.com/gifaldyazkaa/exxyll#%EF%B8%8F-requirements already said, `pnpm` usage is more recommended for this project.

people can use npm / yarn / pnpm / other package managers, but the recommended setup is using pnpm.

Reply:
and those command would installing an pnpm binaries to the local project files, which isn't recommended.